src/query/interpreter.cpp

```diff
@@ -512,7 +518,8 @@ Callback HandleReplicationQuery(ReplicationQuery *repl_query, const Parameters &
     }
 
     case ReplicationQuery::Action::SHOW_REPLICAS: {
-      callback.header = {"name", "socket_address", "sync_mode", "timeout", "state"};
+      callback.header = {"name",    "socket_address",    "sync_mode",
+                         "timeout", "current_timestamp", "number_of_timestamp_behind_master", "state"};
```

Comment:
```suggestion
                         "timeout", "current_timestamp_of_replica", "number_of_timestamp_behind_master", "state"};
```

As you suggested on slack, we should rename this to make it more clear to users.